docs/docs/core/entity-service.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Entity Service
+slug: /entity-service
+tags:
+  - Strapi Service
+  - Entity Service
+---
+
+# Entity Service
+The Entity Service is a wrapper around strapi.db.query
+This wrapper uses the decorator pattern so plugins and strapi core packages can insert decorators to change the output of the entity service function
+Examples are: plugins/i18n and the core/upload
+
+## Checklist writing decorator
```

Comment:
I would call this recommendations, and I would give some code examples. 
The developers using the decorator might want to ignore this even if they are good practices.

Reply:
Marc ignoring those means literally breaking other decorators above or below it

Reply:
and that is for the plugin developer to decide. I would explain why it is a recommendation, and what you might be breaking. But I don't think it's necessary to go that far and say it's "mandatory".

Reply:
I respect your wishes.

But I would still say mandatory since this are contributor docs for strapi aka for strapi maintained plugins so strapi desiding to break other plugins from themselfs would be a bad idea.

Aka what you are basicly telling me is to say sure you can break our I18n plugin with review workflows or your new url signing

But I will make the change to make it recomandations since you asked nicely and it are strapi's docs not mine
@Marc-Roig